### ISSUE TITLE
Accessibility guidance for content

### ISSUE THREAD

**Issue by Tosin-Balogun** [nhsuk/nhsuk-service-manual-backlog#347]
Use this issue to discuss [accessibility guidance for content](https://service-manual.nhs.uk/accessibility/content) in the service manual

**Comment by sarawilcox** [comment]
We're working on an update to [the section on setting page titles](https://service-manual.nhs.uk/accessibility/content#set-page-titles) as it's confusing some people. 

This is the GitHub issue: [Review accessibility guidance on page titles](https://github.com/nhsuk/nhsuk-service-manual/issues/1018).

**Comment by sarawilcox** [comment]
We're also updating [the section on multimedia and videos](https://service-manual.nhs.uk/accessibility/content#make-video-and-multimedia-accessible).

This is the GitHub issue: [Update multimedia section of accessibility guidance (partial)](https://github.com/nhsuk/nhsuk-service-manual/issues/928).

**Comment by sarawilcox** [comment]
Feedback on the PDFs section on this page: 

> I think the section about pdfs could be made a bit clearer where it says: Avoid using PDFs as they are not accessible. If you must use a PDF, make sure the content is also available in HTML form.
> At the moment it's saying avoid using pdfs but then it says if you must use them....This is a bit contradictory. I think it would be worth saying why in some circumstances you may decide to use a pdf.

We have updated the [PDFs section of the content style guide](https://nhsuk-service-manual-dev-uks.azurewebsites.net/content/pdfs-and-other-non-html-documents) but the PDF section of the accessibility guidance is now a bit out of line with it.

**Comment by sarawilcox** [comment]
This is the new section on Making videos and multimedia content accessible: https://service-manual.nhs.uk/accessibility/content#make-video-and-multimedia-accessible.

**Comment by georginaplatt** [comment]
Nikki Drogman/ Senior Brand & Creative Officer/ NHSBT asked question: -

_Do you have any guidance on NHS style and visual guidelines for subtitles please?
I did find a web page recently which detailed the exact colour of the background, but I can’t find the link. Thank you_

Answer from Rich Kelly: -

We have some advice on ['closed captions'](https://service-manual.nhs.uk/accessibility/content#make-video-and-other-multimedia-content-accessible) - which essentially are subtitles. Closed captions means users can turn them on and off and subtitles shouldn't be 'burnt-in' to the video themselves. If you are using YouTube to host video content you don't have any options to change this background colour. It is best to use black background with white text as this offers optimal contrast.
Our multi-media designer is happy to write some guidance to add to the community backlog about background colours for closed captions, but it's important to note that most video hosting providers, i.e. YouTube or vimeo don't give you options to change this.

We do not recommend 'burnt-in' subtitles as they are not pulled in to screenreaders. 

If you insist on making 'open-captions' (burnt-in to the video) then you can use this tool to make sure there is enough contrast between the background colour they choose and text. They will need at least AA status to ensure it's accessible:
[contrast checker](https://contrastchecker.com/)

**Comment by sarawilcox** [comment]
For info only, this is what NHS Digital's corporate style guidelines say: https://digital.nhs.uk/about-nhs-digital/corporate-information-and-documents/nhs-digital-style-guidelines/how-we-look/photography-and-video. See the section on subtitles. 
But see the advice from Rich Kelly above.

**Comment by richkelly13** [comment]
Hi @georginaplatt @sarawilcox
Here's our guidance to this issue.
[Video captions and subtitles colour styling.docx](https://github.com/nhsuk/nhsuk-service-manual-community-backlog/files/6800399/Video.captions.and.subtitles.colour.styling.docx)

**Comment by sarawilcox** [comment]
Sorry it's taken us a while to get to this @richkelly13. I am looking at it now.
